pkg/common/config/util.go

```diff
@@ -26,14 +63,14 @@ func ConfigmapDataCheckSum(configMapData map[string]string) string {
 }
 
 // GetAuthConfigFromKubernetes Looks Up And Returns Kafka Auth ConfigAnd Brokers From Named Secret
-func GetAuthConfigFromKubernetes(ctx context.Context, secretName string, secretNamespace string) (*client.KafkaAuthConfig, error) {
+func GetAuthConfigFromKubernetes(ctx context.Context, secretName string, secretNamespace string) *client.KafkaAuthConfig {
 	secrets := kubeclient.Get(ctx).CoreV1().Secrets(secretNamespace)
 	secret, err := secrets.Get(ctx, secretName, metav1.GetOptions{})
 	if err != nil {
-		return nil, err
+		// Consolidated channel backwards-compatibility - A nonexistent secret returns a nil config
+		return nil
```

Comment:
Kind of weird to mention Consolidated channel here in common code - would just phrase it as "the way this implementation works". `Non-existent Secret results in a Nil KafkaAuthConfig` or something?

Reply:
That's true, especially since it now applies to the distributed channel as well.  I don't think there would be any users who rely on a missing secret to cause an error (but I've been wrong about those kinds of things before).  I made the comment more purpose-oriented.